xamarin/Xamarin.Forms
# [Android] Issue setting Rectangle size

### Description of Change ###

The issue seems to be fixed in 5.0.0. For that reason, this PR only adds a test to validate the issue.

### Issues Resolved ### 

- fixes #12454 

### API Changes ###
 
 None

### Platforms Affected ### 

- Android

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 12454. If can see a Blue Rectangle, the test has passed.

### PR Checklist ###
